test/fixtures.yml

```diff
@@ -641,6 +641,9 @@ nmi:
   login: demo
   password: password
 
+nmi_secure:
+  security_key: '6457Thfj624V5r7WUwc5v6a68Zsd6YEm'
```

Comment:
💬  Comment:

Just in case confirm with David / Alastair if is OK to have that key here in the public repo, seems to be the case, but just to be sure